openff/units/simtk.py

```diff
@@ -7,6 +11,14 @@
 
 from openff.units import unit
 
+__all__ = [
+    "from_simtk",
+    "to_simtk",
+    "simtk_unit_to_string",
+    "string_to_simtk_unit",
+]
```

Comment:
Could you remind me the purpose of these `__all__`? I have a distaste for them because they are really easy to de-sync with the module contents and I don't like star imports.

Reply:
They're there to tell Sphinx what parts of the API to document. There are basically four ways to tell Sphinx what to document:

1. Write out everything you want to document in an RST file (like the Toolkit does). This goes out of sync even faster than `__all__`; at least `__all__` can trip up linters and is in the right file.
2. Treat stuff starting with an underscore as private and document everything else, excluding imported items. This means no re-exports which is basically a nonstarter. I've tried a lot of workarounds but Sphinx is stubborn about this, you can rename imports or assign them to variables and it won't document them if imported items are excluded.
3. Treat stuff starting with an underscore as private and document everything else, including imported items. This means you have to rename everything you import or else it gets documented, and the docs get cluttered because people tend to forget to make things private.
4. Document the stuff in `__all__`, which is Python's standard way to declare stuff as public. At least this way the public API that you're promising not to break is opt-in rather than opt-out.

These all suck but 4 sucks the least. For context, 4 is new; I contributed it to Sphinx because 1-3 weren't workable. I'm not saying that makes it good, but it gives some idea of how frustrating this particular wart has been for me :upside_down_face: